config/Shared/config_default.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/**
+ * This configuration is used for TESTING only and will never be used in production!
+ */
+
+use Generated\Shared\Transfer\MerchantAppOnboardingStatusChangedTransfer;
+use Generated\Shared\Transfer\ReadyForMerchantAppOnboardingTransfer;
+use Ramsey\Uuid\Uuid;
+use Spryker\Shared\AppKernel\AppKernelConstants;
+use Spryker\Shared\Application\ApplicationConstants;
+use Spryker\Shared\Application\Log\Config\SprykerLoggerConfig;
+use Spryker\Shared\ErrorHandler\ErrorHandlerConstants;
+use Spryker\Shared\GlueBackendApiApplication\GlueBackendApiApplicationConstants;
+use Spryker\Shared\GlueJsonApiConvention\GlueJsonApiConventionConstants;
+use Spryker\Shared\Http\HttpConstants;
+use Spryker\Shared\Kernel\KernelConstants;
+use Spryker\Shared\Log\LogConstants;
+use Spryker\Shared\MessageBroker\MessageBrokerConstants;
+use Spryker\Shared\MessageBrokerAws\MessageBrokerAwsConstants;
+use Spryker\Shared\Propel\PropelConstants;
+use Spryker\Shared\ZedRequest\ZedRequestConstants;
+use Spryker\Zed\MessageBrokerAws\MessageBrokerAwsConfig;
+use Spryker\Zed\PropelOrm\Business\Builder\ExtensionObjectBuilder;
+use Spryker\Zed\PropelOrm\Business\Builder\ExtensionQueryBuilder;
+use Spryker\Zed\PropelOrm\Business\Builder\ObjectBuilder;
+use Spryker\Zed\PropelOrm\Business\Builder\QueryBuilder;
+
+// ----------------------------------------------------------------------------
+// ------------------------------ Glue Backend API ----------------------------
+// ----------------------------------------------------------------------------
+$config[GlueBackendApiApplicationConstants::GLUE_BACKEND_API_HOST] = 'api.merchant.local';
+
+$config[KernelConstants::ENABLE_CONTAINER_OVERRIDING] = true;
+$config[KernelConstants::PROJECT_NAMESPACES] =
+$config[GlueBackendApiApplicationConstants::PROJECT_NAMESPACES] = [
+    'Spryker',
+];
+$config[ZedRequestConstants::ZED_API_SSL_ENABLED] = (bool)getenv('SPRYKER_ZED_SSL_ENABLED');
+
+$config[ApplicationConstants::BASE_URL_ZED] = sprintf(
+    'https://%s',
+    'api.merchanr.local',
+);
+
+$config[AppKernelConstants::APP_IDENTIFIER] = Uuid::uuid4()->toString();
+
+$config[HttpConstants::URI_SIGNER_SECRET_KEY] = Uuid::uuid4()->toString();
+
+$config[GlueJsonApiConventionConstants::GLUE_DOMAIN] = sprintf(
+    '%s://%s',
+    getenv('SPRYKER_SSL_ENABLE') ? 'https' : 'http',
+    $config[GlueBackendApiApplicationConstants::GLUE_BACKEND_API_HOST] ?: 'localhost',
+);
+
+$config[MessageBrokerConstants::CHANNEL_TO_SENDER_TRANSPORT_MAP] = [
+    'app-events' => MessageBrokerAwsConfig::HTTP_CHANNEL_TRANSPORT,
+];
```

Comment:
MessageBroker config has to be in README file, so an app developer knows what to add to the app config.

Reply:
Added